FlowCrypt/Functionality/DataManager/Encrypted Storage/EncryptedStorage.swift

```diff
@@ -272,6 +272,18 @@ extension EncryptedStorage {
 }
 
 extension EncryptedStorage {
+    static var path: String {
+        get throws {
+            try getDocumentDirectory() + "/" + Constants.encryptedDbFilename
+        }
```

Comment:
Is `Constants.encryptedDbFilename` only used here? If yes, you can remove that variable and write `+ "/encrypted.realm"`